Emby.Server.Implementations/Services/HttpContextExtension.cs

```diff
@@ -0,0 +1,33 @@
+using MediaBrowser.Model.Services;
+using Microsoft.AspNetCore.Http;
+
+namespace Emby.Server.Implementations.Services
+{
+    /// <summary>
+    /// Extention to enable the service stack request to be stored in the HttpRequest object.
+    /// </summary>
+    public static class HttpContextExtension
+    {
+        private const string SERVICESTACKREQUEST = "ServiceRequestStack";
+
+        /// <summary>
+        /// Set the service stack request.
```

Comment:
It might be better to write `ServiceStack` this way to make it clear you're referring to the framework, since that is how they write it

```suggestion
        /// Set the ServiceStack request.
```